app/views/recipes/_navmenu.html.erb

```diff
@@ -0,0 +1,20 @@
+<div class="nav_menu">
+  <div class="nav_navigation">
+    <%= link_to publicrecipes_path, class:"nav_items" do %>
+      <p>Public Recipes</p>
+    <% end %>
+    <%= link_to recipes_path, class:"nav_items" do %>
+      <p>My Recipes</p>
+    <% end %>
+    <%= link_to foods_path, class:"nav_items" do %>
+      <p> My food list </p>
+    <% end %>
+    <%= link_to shoppinglist_index_path, class:"nav_items" do %>
+      <p> General Shopping List </p>
+    <% end %>
+  </div>
+  <div class="nav_user">
+    <p class="user_logged"><%= current_user.name %></p>
+    <%= button_to "Sign out", destroy_user_session_path, method: :delete %>
+  </div>
```

Comment:
- [ ] I'm getting an error message around this part of your code. Kindly try to fix it.

<img width="1098" alt="Screen Shot 2022-10-16 at 06 37 35" src="https://user-images.githubusercontent.com/69011963/196013838-465194c4-99ad-47ba-a8b0-3cacffe17939.png">